test/test_utils/__init__.py

```diff
@@ -324,6 +331,47 @@ def get_dlc_images():
     raise RuntimeError(f"Cannot find any images for in {test_images}")
 
 
+def parse_canary_images(framework, region):
+    tf1 = "1.15"
+    tf1_long = "1.15.2"
+    tf2 = "2.2"
+    mx = "1.6"
+    pt = "1.5"
+
+    if framework == "tensorflow":
+        framework = "tensorflow2" if "tensorflow2" in os.getenv("CODEBUILD_BUILD_ID") else "tensorflow1"
+
+    images = {
+        "tensorflow1":
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-gpu-py3 "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-cpu-py3 "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-cpu-py2 "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-gpu-py2 "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-inference:{tf1}-gpu "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-inference:{tf1}-cpu "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-inference:{tf1_long}-gpu-py27-cu100-ubuntu18.04 "
```

Comment:
Are all these images supported in SM?

Reply:
they are not, but I don't think any TF py2 inference images are

Reply:
It should be alright to remove non-SM images from this list.